app/models/miq_report.rb

```diff
@@ -79,15 +79,7 @@ def self.having_report_results(options = {})
     miq_group_ids = options[:miq_groups].collect(&:id) unless options[:miq_groups].nil?
 
     miq_group_ids ||= options[:miq_group_ids]
-    q = joins(:miq_report_results).merge(MiqReportResult.for_groups(miq_group_ids)).distinct
-
-    if options[:select]
-      cols = Array.wrap(options[:select])
-      cols = cols.dup.unshift(:id) unless cols.include?(:id)
-      cols.each { |c| q = q.select(arel_table[c]) }
```

Comment:
`arel_table` would not work with virtual attributes. I'm surprised that this didn't change ages ago.

I'm sad to see this go but I don't think this code has anything for us anymore.
It is a TODO that we will probably never address